### PR TITLE
Get mp4s rendering and playing on editorial page

### PR DESCRIPTION
**Pull request by Justin-Holmes**
https://www.pivotaltracker.com/story/show/155603725

### CODE REVIEW

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a linked tracker story for this PR in the description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Justin-Holmes** [comment]
I still need to get tests passing and cover images rendering gifs properly again.

File: src/pages/ArtistInvitesPage.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { selectQueryPreview } from '../selectors/routing'
 import { selectRandomArtistInvitePromotion } from '../selectors/promotions'
 import { selectJson } from '../selectors/store'
 import { css } from '../styles/jss'
-import { selectDPI } from '../selectors/gui'
+import { selectHeroDPI } from '../selectors/gui'
```
Comment (gblakeman) [inline src/pages/ArtistInvitesPage.js]: @Justin-Holmes Did you intend to switch these out for the Artist Invites? This is what’s tripping your build.

Comment (Justin-Holmes) [reply src/pages/ArtistInvitesPage.js]: @gblakeman Yep forgot to update this is in `mapStateToProps`.

**Pull request merged**